### PR TITLE
Host mission control

### PR DESCRIPTION
**Pull request by aninternetof**
Moves mission control from other repo to this one. Fixes tabs/spaces. Moves from Foundation to bootstrap.

I'd reference what issues this closes, but I still need to move the issues over from the other repo.

### CODE REVIEW

File: mission_control/static/css/one-page.css [hunk 1]
```diff
@@ -0,0 +1,91 @@
+body{
+	margin: 0;
```
Comment (cabarnes) [inline mission_control/static/css/one-page.css]: Inconsistent whitespace throughout this file. Maybe mixed tabs and spaces?

File: mission_control/static/js/blockly-api.js [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+			console.log('Here it is: ' + eventQueue);
+			currentEvent = eventQueue.shift();
+		}
+		return interpreter.createPrimitive(currentEvent);
+	};
+	interpreter.setProperty(scope, 'popEventQueue',
+		interpreter.createNativeFunction(wrapper));
+}
+
+// function checkSensor(sensor) {
```
Comment (cabarnes) [inline mission_control/static/js/blockly-api.js]: Is any of this code needed anymore?

File: mission_control/static/js/rover-api.js [hunk 3]
```diff
@@ -0,0 +1,141 @@
...
+    $.get(roverResource('blockdiagrams'), function(json){
+      var duplicate = json.result.indexOf(designName) > -1;
+      if (duplicate) {
+        $('#nameErrorArea').text('This name has already been chosen. Please pick another one.');
+      } else {
+        saveDesign();
+        $('#nameErrorArea').empty();
+        $('a#designNameArea').text(designName);
+        $('a#downloadLink').attr("onclick", "return downloadDesign(\""+designName+".xml\")");
+        $('#nameModal').foundation('reveal', 'close');
```
Comment (cabarnes) [inline mission_control/static/js/rover-api.js]: Is this a foundation function that needs to be changed?

File: mission_control/templates/home.html [hunk 4]
```diff
@@ -0,0 +1,299 @@
...
+    <category name="variables" colour="20" custom="VARIABLE">
+    </category>
+    <category name="functions" colour="80" custom="PROCEDURE">
+    </category>
+  </xml>
+
+  <script src="{% static 'js/mission-control.js' %}"></script>
+
+  <script type="text/javascript">
+  $(document).foundation();
```
Comment (cabarnes) [inline mission_control/templates/home.html]: Foundation should be removed.

File: mission_control/static/css/one-page.css (hunk 1 above)
Comment (aninternetof) [inline mission_control/static/css/one-page.css]: Whoops. Yes, fixing.

File: mission_control/static/js/blockly-api.js (hunk 2 above)
Comment (aninternetof) [inline mission_control/static/js/blockly-api.js]: Nope, deleting.

File: mission_control/static/js/rover-api.js (hunk 3 above)
Comment (aninternetof) [inline mission_control/static/js/rover-api.js]: Yes. Fixing.

File: mission_control/templates/home.html (hunk 4 above)
Comment (aninternetof) [inline mission_control/templates/home.html]: Yes.

**Comment by aninternetof** [comment]
How's it look now?

**Comment by cabarnes** [comment]
There are still some foundation libraries in `mission_control/static/js` and `mission_control/static/css`.
Also, the change in `mission_control/static/js/blockly-api.js` with the dead code removal doesn't seem to be included.

**Comment by aninternetof** [comment]
Ok, I think I've really got it now!

**Pull request merged**